### PR TITLE
opam 2.0.1 annoucement

### PR DESCRIPTION
**Pull request by rjbou**
ping @altgr

### CODE REVIEW

File: opam-2-0-1.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+--BODY--
+
+We are pleased to announce the release of [opam 2.0.1](https://github.com/ocaml/opam/releases/tag/2.0.1).
+
+This new version contains mainly fixes, some platform-specific:
+- Cold boot for MacOS/CentOS/Alpine
+- Install checksum validation on MacOS
+- Archive extraction for OpenBSD, using `gtar`
+- Update mccs for MacOS/Nix
+- Remove Gnu sed option for OpenBSD/FreeBSD
```
Comment (avsm) [inline opam-2-0-1.md]:
```suggestion
- Do not use GNU-sed specific features in the release Makefile, to fix build on OpenBSD/FreeBSD
```

File: opam-2-0-1.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+date: "2018-10-24"
+--BODY--
+
+We are pleased to announce the release of [opam 2.0.1](https://github.com/ocaml/opam/releases/tag/2.0.1).
+
+This new version contains mainly fixes, some platform-specific:
+- Cold boot for MacOS/CentOS/Alpine
+- Install checksum validation on MacOS
+- Archive extraction for OpenBSD, using `gtar`
+- Update mccs for MacOS/Nix
```
Comment (avsm) [inline opam-2-0-1.md]:
```suggestion
- Fix compilation of mccs on MacOS and Nix platforms
```

It's not clear to me what the Nix fix was... the PR only affects macOS in ocaml-mccs

File: opam-2-0-1.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+]
+date: "2018-10-24"
+--BODY--
+
+We are pleased to announce the release of [opam 2.0.1](https://github.com/ocaml/opam/releases/tag/2.0.1).
+
+This new version contains mainly fixes, some platform-specific:
+- Cold boot for MacOS/CentOS/Alpine
+- Install checksum validation on MacOS
+- Archive extraction for OpenBSD, using `gtar`
```
Comment (avsm) [inline opam-2-0-1.md]:
```suggestion
- Archive extraction for OpenBSD now defaults to using `gtar`
```

File: opam-2-0-1.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+authors: [
+  "Raja Boujbel - OCamlPro" {"mailto:raja.boujbel(à)ocamlpro.com"}
+  "Louis Gesbert - OCamlPro" {"mailto:louis.gesbert(à)ocamlpro.com"}
+]
+date: "2018-10-24"
+--BODY--
+
+We are pleased to announce the release of [opam 2.0.1](https://github.com/ocaml/opam/releases/tag/2.0.1).
+
+This new version contains mainly fixes, some platform-specific:
```
Comment (avsm) [inline opam-2-0-1.md]:
```suggestion
This new version contains mainly [backported fixes](https://github.com/ocaml/opam/pull/3560), some platform-specific:
```

File: opam-2-0-1.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+- Cold boot for MacOS/CentOS/Alpine
+- Install checksum validation on MacOS
+- Archive extraction for OpenBSD, using `gtar`
+- Update mccs for MacOS/Nix
+- Remove Gnu sed option for OpenBSD/FreeBSD
+- Cleaning to enable reproducible builds
+- Update configure scripts
+
+And some opam specific:
+- git: fix git fetch by sha1
```
Comment (avsm) [inline opam-2-0-1.md]: what's the impact of this change? does it affect all users?

**Comment by AltGr** [comment]
LGTM

File: opam-2-0-1.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- linting: add `test` variable warning and empty description error
+- upgrade: convert pinned but not installed opam files
+- error reporting: more comprehensible error for tar extraction, and upgrade of git-url compilers
+- opam show: upgrade given local files
+- list: add `--silent` option for a silent output and returns 1 if list is empty
```
Comment (avsm) [inline opam-2-0-1.md]: It's worth noting that this affects the CLI backwards compatibility with scripts.

File: opam-2-0-1.md (hunk 2 above)
Comment (rjbou) [reply opam-2-0-1.md]: The issue origin is NixOS packaging
